### PR TITLE
FIX: hack directory now lives inside calico subdirectory

### PR DESCRIPTION
**Pull request by james-pollard-leapyear**
## Description

Several links in docs refer to the hack directory, which changed locations in the repo from `./hack` to `calico/hack`

## Todos

~- [ ] Tests~
~- [ ] Documentation~
~- [ ] Release note~

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by james-pollard-leapyear** [comment]
@marvin-tigera anything else for me/us to do here?

**Review by fasaxc — APPROVED** [review]
Expect these files will move again as we tidy up the new repo but thanks for the clean-up!

File: calico/reference/etcd-rbac/certificate-generation.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Authority (CA), Certificates and Keys that can be used to authenticate a
 specific user with etcd. There are many different tools that can be used to
 generate these files. This tutorial tries to layout the unique or specific
 details that are needed for each of the different certificates but uses the
-[hack/tls-setup tool from the etcd repo](https://github.com/coreos/etcd/tree/master/hack/tls-setup){:target="_blank"},
+[hack/tls-setup tool from the etcd repo](https://github.com/coreos/etcd/tree/master/calico/hack/tls-setup){:target="_blank"},
```
Comment (fasaxc) [inline calico/reference/etcd-rbac/certificate-generation.md:14]: Spoke too soon, this one is a different repo

Comment (james-pollard-leapyear) [reply calico/reference/etcd-rbac/certificate-generation.md]: @fasaxc Great catch - fixed!

File: calico/reference/etcd-rbac/certificate-generation.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Authority (CA), Certificates and Keys that can be used to authenticate a
 specific user with etcd. There are many different tools that can be used to
 generate these files. This tutorial tries to layout the unique or specific
 details that are needed for each of the different certificates but uses the
-[hack/tls-setup tool from the etcd repo](https://github.com/coreos/etcd/tree/master/hack/tls-setup){:target="_blank"},
+[hack/tls-setup tool from the etcd repo](https://github.com/coreos/etcd/tree/master/calico/tls-setup){:target="_blank"},
```
Comment (caseydavenport) [inline calico/reference/etcd-rbac/certificate-generation.md:14]: I think this link is still not quite right - it should be the original - the new link is a 404.

Comment (james-pollard-leapyear) [reply calico/reference/etcd-rbac/certificate-generation.md]: thanks - fixed! Should be ready for merge now @caseydavenport @fasaxc

**Review by fasaxc — APPROVED** [review]
LGTM

**Pull request merged**